### ISSUE TITLE
Show module returns Parser Output is empty

### ISSUE THREAD

**Issue by erickavila888** [CiscoTestAutomation/genieparser#284]
I get the Parser Output is empty when I attempt to parse show module on my 4k switch. I made sure both pyats and genie libraries are updated.

Switch version:
```
Cisco IOS Software, Catalyst 4500 L3 Switch Software (cat4500-IPBASEK9-M), Version 15.0(2)SG8, RELEASE SOFTWARE (fc2)
Technical Support: http://www.cisco.com/techsupport
Copyright (c) 1986-2013 by Cisco Systems, Inc.
Compiled Mon 02-Dec-13 17:00 by prod_rel_team
Image text-base: 0x10000000, data-base: 0x12095E08

ROM: 12.2(31r)SGA
Pod Revision 14, Force Revision 31, Tie Revision 32

3.3-hil-bw-2 uptime is 5 weeks, 5 days, 21 hours, 5 minutes
System returned to ROM by power-on
System image file is "bootflash:cat4500-ipbasek9-mz.150-2.SG8.bin"


This product contains cryptographic features and is subject to United
States and local country laws governing import, export, transfer and
use. Delivery of Cisco cryptographic products does not imply
third-party authority to import, export, distribute or use encryption.
Importers, exporters, distributors and users are responsible for
compliance with U.S. and local country laws. By using this product you
agree to comply with applicable laws and regulations. If you are unable
to comply with U.S. and local laws, return this product immediately.

A summary of U.S. laws governing Cisco cryptographic products may be found at:
http://www.cisco.com/wwl/export/crypto/tool/stqrg.html

If you require further assistance please contact us by sending email to
export@cisco.com.

cisco WS-C4506 (MPC8540) processor (revision 11) with 524288K bytes of memory.
Processor board ID FOX0712003Y
MPC8540 CPU at 800Mhz, Supervisor V-10GE
Last reset from PowerUp
2 Virtual Ethernet interfaces
244 Gigabit Ethernet interfaces
2 Ten Gigabit Ethernet interfaces
511K bytes of non-volatile configuration memory.

Configuration register is 0x2102
```

testbed object excerpt:
```yaml
devices:
  3.3-hil-bw-2:
    type: cat4k
    series: cat4k
    platform: cat4500
    os: ios
    alias: '3.3'
    connections:
      defaults:
        class: 'unicon.Unicon'
      vty:
        protocol: ssh
        ip: 10.0.3.3
    custom:
      vtpdomain: hilldistro
```

IPython output:
```
In [3]: mod = hil.parse('show module')

2020-10-05 09:41:12,607: %UNICON-INFO: +++ 3.3-hil-bw-2: executing command 'show module' +++
show module
Chassis Type : WS-C4506

Power consumed by backplane : 0 Watts

Mod Ports Card Type                              Model              Serial No.
---+-----+--------------------------------------+------------------+-----------
 1     6  Sup V-10GE 10GE (X2), 1000BaseX (SFP)  WS-X4516-10GE      JAE1122K5VJ
 2    48  10/100/1000BaseT (RJ45)V, Cisco/IEEE   WS-X4548-GB-RJ45V  JAE1341L348
 3    48  10/100/1000BaseT (RJ45)V, Cisco/IEEE   WS-X4548-GB-RJ45V  JAE1344MGM7
 4    48  10/100/1000BaseT (RJ45)V, Cisco/IEEE   WS-X4548-GB-RJ45V  JAE1340K3D8
 5    48  10/100/1000BaseT (RJ45)V, Cisco/IEEE   WS-X4548-GB-RJ45V  JAE1343LYYG
 6    48  10/100/1000BaseT (RJ45)V, Cisco/IEEE   WS-X4548-GB-RJ45V  JAE10212ZV4

 M MAC addresses                    Hw  Fw           Sw               Status
--+--------------------------------+---+------------+----------------+---------
 1 000c.ce94.b640 to 000c.ce94.b645 3.4 12.2(31r)SGA 15.0(2)SG8       Ok
 2 0027.0d6c.d900 to 0027.0d6c.d92f 4.1                               Ok
 3 0026.9951.ade0 to 0026.9951.ae0f 4.1                               Ok
 4 0027.0d6d.0630 to 0027.0d6d.065f 4.1                               Ok
 5 0026.9951.4a20 to 0026.9951.4a4f 4.1                               Ok
 6 0017.e09f.aa00 to 0017.e09f.aa2f 3.0                               Ok

3.3-hil-bw-2#
---------------------------------------------------------------------------
SchemaEmptyParserError                    Traceback (most recent call last)
<ipython-input-3-d079886cec05> in <module>
----> 1 mod = hil.parse('show module')

~/pyats/lib64/python3.6/site-packages/genie/conf/base/device.cpython-36m-x86_64-linux-gnu.so in genie.conf.base.device.Device.parse()

~/pyats/lib64/python3.6/site-packages/genie/conf/base/device.cpython-36m-x86_64-linux-gnu.so in genie.conf.base.device.Device._get_parser_output()

~/pyats/lib64/python3.6/site-packages/genie/metaparser/_metaparser.cpython-36m-x86_64-linux-gnu.so in genie.metaparser._metaparser.MetaParser.parse()

~/pyats/lib64/python3.6/site-packages/genie/metaparser/_metaparser.cpython-36m-x86_64-linux-gnu.so in genie.metaparser._metaparser.MetaParser.parse()

~/pyats/lib64/python3.6/site-packages/genie/metaparser/util/schemaengine.cpython-36m-x86_64-linux-gnu.so in genie.metaparser.util.schemaengine.Schema.validate()

SchemaEmptyParserError: Parser Output is empty
```

**Comment by tahigash** [comment]
I've identified the issue. Now working on the fix.

**Comment by tahigash** [comment]
This is fixed and fix will be coming with next release v20.10 (end of October).

The problem was, parser was missing for `ios`, but exists for `iosxe`.
as workaround now, below updated yaml should work.

```
devices:
  3.3-hil-bw-2:
    type: cat4k
    series: cat4k
    platform: cat4k # <<< change from `cat4500`
    os: iosxe # <<< change from `ios`
    alias: '3.3'
    connections:
      defaults:
        class: 'unicon.Unicon'
      vty:
        protocol: ssh
        ip: 10.0.3.3
    custom:
      vtpdomain: hilldistro
      abstraction:    # added
        order:        # Addedd
          - os        # Added
          - platform  # Added
```
After v20.10, you can change back the os to `ios`. but keep `platform` and `custom`.

Closing the issue, but please feel free to reopen if any issue with above.
